examples/wCCD/src/wCCD.tsx

```diff
@@ -138,25 +146,29 @@ export default function wCCD(props: WalletConnectionProps) {
     const mainnet = 'mainnet';
     let WCCD_CONTRACT_INDEX: bigint;
 
+    let grpcClient: ConcordiumGRPCClient | undefined;
+
     if (process.env.NETWORK === mainnet) {
         WCCD_CONTRACT_INDEX = WCCD_CONTRACT_INDEX_MAINNET;
```

Comment:
Not a fan of assigning what looks like a constant. Would prefer constructing a configuration object (that also included the `Network` object).